src/Layouts/Partials/TransformPanel.vala

```diff
@@ -189,6 +189,14 @@ public class Akira.Layouts.Partials.TransformPanel : Gtk.Grid {
         }
 
         selected_item = selected_items.nth_data (0);
+
+        if (selected_item is Lib.Models.CanvasArtboard) {
+            hflip_button.hide ();
+            vflip_button.hide ();
+        } else {
+            hflip_button.show ();
+            vflip_button.show ();
+        }
```

Comment:
Thanks for the contribution.
I prefer to change the button sensitivity instead of hiding them.
Already the Rotation field stays insensitive, so we keep consistency and we prevent sudden changes in the UI by hiding items that we never hide.

Reply:
Hi, thanks for the quick reply!
I see, I will try to change the sensitivity instead then. 

> This PR should also take care of stopping the flip command if an Artboard is selected

Sorry about that, I have not even realized there was a shortcut for it

Reply:
No need to apologize, it's a big code base and the simple fact that you're willing to contribute to it is already amazing.
I'm here to help you figure out all the nits of the source code.
Thank you so much :smile:

Reply:
Awesome, thank you a lot :) I have made some changes again, hope the PR will be updated soon.
By the way, do you have a mailing list/IRC/discord for this project?